### ISSUE TITLE
WCAG 2.2 errata: success criteria 2.4.13

### ISSUE THREAD

**Issue by rscano** [w3c/wcag#3499]
In SC the "focus indicator" has a title attribute "[New]" instead of "pixels that are changed to visually indicate when a user interface component is in a focused state"

in same SC also title attribute for permeter has same issue. Should be title="continuous line forming the boundary of a shape not including shared pixels, or the minimum bounding box, whichever is shortest."

**Comment by mbgower** [comment]
Since https://github.com/w3c/wcag/issues/3511 captures the underlying issue, and this is simply another example of the outcome, I am closing this as a duplicate.
